src/csharp/Microsoft.Spark.E2ETest/IpcTests/Sql/SparkSessionTests.cs

```diff
@@ -78,13 +79,14 @@ public void TestCreateDataFrame()
             // Calling CreateDataFrame with schema
             {
                 var data = new List<GenericRow>();
-                data.Add(new GenericRow(new object[] { "Alice", 20 }));
-                data.Add(new GenericRow(new object[] { "Bob", 30 }));
+                data.Add(new GenericRow(new object[] { "Alice", 20, new DateTime(2020, 1, 1) }));
+                data.Add(new GenericRow(new object[] { "Bob", 30, new DateTime(2020, 1, 2) }));
 
                 var schema = new StructType(new List<StructField>()
                 {
                     new StructField("Name", new StringType()),
-                    new StructField("Age", new IntegerType())
+                    new StructField("Age", new IntegerType()),
+                    new StructField("Date", new DateType())
```

Comment:
can we also add a test that just tests `IEnumerable<DateTime>`?

Reply:
Sure, I have added the test.